packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -542,4 +557,14 @@ class _SummaryCardState extends State<SummaryCard> {
         await robotoffInsightHelper.haveInsightAnnotationsVoted(questions);
     return questions;
   }
+
+  void _showNotImplemented(BuildContext context) {
```

Comment:
I don't think moving that code was particularly important. But it takes me additional time to check the differences and so on. If possible, don't move code :)

Reply:
Yes I pay a bit more attention on it from now on, it confused me a bit  between all the widgets